salsa_dancing_molecules/startup/generate_json.py

```diff
@@ -0,0 +1,58 @@
+"""Module converts the configurations into a Json-files."""
+import json
+
+
+def config_to_configs(config):
+    """Generate all combinations of elements from dict with lists.
+
+    Args:
+        config - dictionary to be used as template.
+    Returns:
+        list_of_json_dicts - list of all the different new dictionaries.
+    """
+    list_of_json_dicts = []
+    for key in config.keys():
```

Comment:
This code is not easy to understand. Would it be possible to add a few comments or factor it into more functions with function names that explain what's going on?